templates/default/less/variables.less

```diff
@@ -657,7 +657,7 @@
 @il-item-padding: 20px;
 @il-item-marker-border-left: 5px solid;
 @il-item-h5-margin: 4px 0px 2px 0px;
-
+@il-item-group-max-width: 997px;
```

Comment:
How do we come to this magic 997? I guess this related to the container desktop:
 ```
.ilContainerWidth {
	max-width: 1400px;
}
```

I think if we start using such global sizes to define new max-width we should introduce the max-width of the container as variable an make transparent how this size depends from the overall container ones.

I guess this would be then:
```
//Max width of the main content area, including tabs breadcrumb etc.
@il-main-content-max-width: 1400px - @grid-gutter-width; 
//Max width of the right sidebar, note the sidebar uses 3 columns, 342.5px
@il-sidebar-max-width: @il-main-content-max-width/@grid-columns * 3
//Max width of the right sidebar, 997.55px
@il-item-group-max-width: @il-main-content-max-width-@il-sidebar-max-width-@grid-gutter-width;
```